zerver/models/alert_words.py

```diff
@@ -15,25 +15,25 @@ class AlertWord(models.Model):
     # Realm isn't necessary, but it's a nice denormalization.  Users
     # never move to another realm, so it's static, and having Realm
     # here optimizes the main query on this table, which is fetching
-    # all the alert words in a realm.
+    # all the watched phrases in a realm.
     realm = models.ForeignKey(Realm, db_index=True, on_delete=CASCADE)
     user_profile = models.ForeignKey(UserProfile, on_delete=CASCADE)
     # Case-insensitive name for the alert word.
-    word = models.TextField()
+    watched_phrase = models.TextField()
```

Comment:
Should we call it just `phrase` ?

`phrase` field of `WatchedPhrase` model. It's clear enough, I think.